tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -374,16 +374,15 @@ module.exports = {
         .waitForElementNotPresent('@loadingIndicator')
         .waitForElementPresent(this.getFileRowSelectorByFileName(element))
     },
-    assertSharingIsDisabled: function () {
+    assertSharingIsDisabled: function (resource) {
+      const fileRow = this.getFileRowSelectorByFileName(resource)
       return this
         .closeSidebar(100)
-        .assert
-        .elementNotPresent('@shareButtonInFileRow')
-        .waitForElementVisible('@firstFileRow')
         .useXpath()
-        .click('@firstFileRow')
-        .assert
-        .elementNotPresent('@sidebarSharingContent')
+        .assert.elementNotPresent('@shareButtonInFileRow')
```

Comment:
`@shareButtonInFileRow` selects any share button in one of the elements in the list. But I think it is a better idea to specifically select the share button of the file row we are checking.